openedx/adg/lms/webinar/admin.py

```diff
@@ -0,0 +1,25 @@
+"""
+Registering models for webinar app.
+"""
+from django.contrib import admin
+
+from .models import Webinar
+
+
+@admin.register(Webinar)
+class WebinarAdmin(admin.ModelAdmin):
+    """
+    Model admin for webinar
+    """
+
+    list_display = ('title', 'subject', 'start_time', 'presenter', 'status', )
+    list_filter = ('start_time', 'presenter', 'status', 'language', )
```

Comment:
Please remove the presenter as it is showing all the users in the filter in Django admin and having 30k+ users this does not make any sense.